pkg/odo/cli/login/login.go

```diff
@@ -86,5 +99,6 @@ func NewCmdLogin(name, fullName string) *cobra.Command {
 	loginCmd.Flags().StringVarP(&o.tokenFlag, "token", "t", "", "token, will prompt if not provided")
 	loginCmd.Flags().BoolVar(&o.skipTlsFlag, "insecure-skip-tls-verify", false, "If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure")
 	loginCmd.Flags().StringVar(&o.caAuthFlag, "certificate-authority", "", "Path to a cert file for the certificate authority")
+	loginCmd.Flags().StringVar(&o.serverFlag, "server", "", "OpenShift server to log into")
```

Comment:
```suggestion
	loginCmd.Flags().StringVar(&o.serverFlag, "server", "", "OpenShift/Kubernetes Server host to log into")
```

Reply:
I didn't put Kubernetes because I don't remember logging into vanilla Kubernetes using any kind of login command. If you see, `kubectl` doesn't have a `login` command to it. This is OpenShift specific thingy, IMO.